Makefile

```diff
@@ -23,6 +23,9 @@ test: fmt vet
 integ-test:
 	go test -v -test.timeout=5m ./tests/integration/... -coverprofile cover.out
 
+test-auth:
```

Comment:
I don't like these polluting the Makefile, so I created alternative approach that allows defining the parameters in the make command in the PR #398